README.md

```diff
@@ -19,6 +19,12 @@ Follow the instructions on [get started with GitHub Pages](//pages.github.com/).
 
 Fork [reveal-jekyll](//github.com/tasmo/reveal-jekyll) and name it with your user or organisation name like `<yourname>.github.io`.
 
+You can also host as a [project page](https://help.github.com/articles/user-organization-and-project-pages/#project-pages):
+
+- Clone this repo with a name like `mytalk`
+- Delete the `gh-pages` branch, create a new `gh-pages` branch off of `master`, and set `gh-pages` as the default branch. You can do all this through the Github web UI (TODO: details)
```

Comment:
You know, I'm afraid the branch swapping is going to scare people away. I'm thinking to just leave that part out, and let people start with the current `gh-pages` branch, which only differs by having different demo content. Does that seem ok?